scripts/node-globals.js

```diff
@@ -0,0 +1 @@
+export const { Buffer } = require('buffer')
```

Comment:
ditto for `"buffer"` dependency - I'm assuming we're supporting browser builds here too